src/variables.js

```diff
@@ -117,7 +117,7 @@ export function createVariable(workspace, opt_callback, opt_type) {
 
         var flyout = workspace.isFlyout ? workspace : workspace.getFlyout();
         var variableBlockId = variable.getId();
-        workspace.getToolbox().runAfterRerender(() => {
+        workspace.getToolbox()?.runAfterRerender(() => {
```

Comment:
When does this happen with no toolbox?

Reply:
I didn't root cause this, but it was null when adding a new broadcast message from the dropdown menu item in the flyout.